mmcls/core/visualization/image.py

```diff
@@ -247,11 +244,10 @@ def put_img_infos(self,
         width, height = img.shape[1], img.shape[0]
         img = np.ascontiguousarray(img)
 
-        # add a small EPS to avoid precision lost due to matplotlib's
+        # there might be a 1px precision lost due to matplotlib's
         # truncation (https://github.com/matplotlib/matplotlib/issues/15363)
         dpi = self.fig_save.get_dpi()
-        self.fig_save.set_size_inches((width + EPS) / dpi,
-                                      (height + EPS) / dpi)
+        self.fig_save.set_size_inches(width / dpi, height / dpi)
```

Comment:
Let us reserve the `EPS` because it can avoid the precision lost in most situations.